.github/workflows/ci-nightly.yaml

```diff
@@ -0,0 +1,29 @@
+name: CI (Nightly)
+
+on:
+  schedule:
+    - cron: '30 3 * * *'
+  workflow_dispatch:
+
+jobs:
+  create-timestamp:
+    uses: zingolabs/zingo-mobile/.github/workflows/create-timestamp.yaml@dev
+
+  create-cache-key:
+    uses: zingolabs/zingo-mobile/.github/workflows/create-cache-key.yaml@dev
+
+  build-android:
+    uses: zingolabs/zingo-mobile/.github/workflows/build.yaml@dev
+    needs: create-cache-key
+    with:
+      cache-key: ${{ needs.create-cache-key.outputs.cache-key }}
+      arch: armeabi-v7a
+
+  integration-test-android:
+    uses: zingolabs/zingo-mobile/.github/workflows/integration-test.yaml@dev
+    needs: [ create-timestamp, create-cache-key, build-android ]
+    with:
+      timestamp: ${{ needs.create-timestamp.outputs.timestamp }}
+      cache-key: ${{ needs.create-cache-key.outputs.cache-key }}
+      abi: armeabi-v7a
```

Comment:
in CI nightly only for one arch? I don't know if can be a good idea to do for the 4 archs...????

Reply:
i can add all four archs good idea